.gitignore

```diff
@@ -1,13 +1,14 @@
-build/unix/*.eo
-build/unix/*.o
-build/unix/pfdicdat.h
-build/unix/pforth
-build/unix/pforth.dic
-build/unix/pforth_standalone
-build/win32/**/.vs
-build/win32/**/Debug
-build/win32/**/Release
+platform/unix/*.eo
```

Comment:
I like changing build to platform.
The use of "build" was a very old decision and wrong in hindsight.